input/pagecontent/payload.md

```diff
@@ -1,7 +1,106 @@
+
+<figure>
+    <img src="GenomeX_Data_Exchange_Bundle_Types.jpg"
+         alt="Data Exchange bundle types">
+    <figcaption>Figure 2. Data Exchange bundle types</figcaption>
+</figure>
+
+The recommended path to enable a smooth transition involves multiple steps to bridge from HL7v2 only ordering and resulting systems to a fully FHIR-ized system. To that end, these recommendations are provided:
+- Continue to embed the final PDF test report in the HL7v2 message
+- Begin to embed a test result FHIR transaction bundle in the HL7v2 message (option 1)
+	- Option to embed the final PDF test report in FHIR bundle to validate full system operation prior to moving off HL7v2 transport 
+- Introduce the FHIR RP once FHIR servers are exposed for the lab and ordering organization (option 2) 
+	- RP to a FHIR transaction bundle 
+	- RP to a FHIR searchset
+
+#### Decision: CodeableCocept Usage Pattern
+Include codeableConcept AND represent the display text in the text field as the GenomeX standard for coded system data inputs.
+
+The text field is used to represent an alternate text based value that is understandable for clinical use in addition to the coded value. For example, just displaying "c.2251A>T" versus "NM_001127208.2:c.2251A>T" as several members currently provide.
+
+GenomeX participants agree to the proper coded representation using a codeableConcept and represent the alternate text based value in the text field. During the early phases of development, the "text only" option below is acceptable, but not preferred, as members enhance their workflows to include the full coded representation. This may not be an acceptable option in all cases, such as where the coded system is required.
+
+The following three values provide an example:
+
+```
+{
+	"valueCodeableConcept": {
+		"text": "c.2251A>T" 
+	}
+}, 
+{
+	"valueCodeableConcept": {
+		"coding": [ 
+			{ 
+				"system": "http://varnomen.hgvs.org",
+				"code": "NM_001127208.2:c.2251A>T",
+				"display": "NM_001127208.2:c.2251A>T 
+			} 
+		] 
+	}
+},
+{ 
+   "valueCodeableConcept": {
+	    "coding": [ 
+		    { 
+		         "system": "http://varnomen.hgvs.org",
+		         "code": "NM_001127208.2:c.2251A>T",
+		         "display": "NM_001127208.2:c.2251A>T 
+			} 
+		],
+		"text": "c.2251A>T" 
+	}
+}
+```
+
+### Implementation / Validation Venues
+The decisions and consensus reached by the Data Exchange community through the process described below have been tested and validated during multiple connectathons, specifically May 2023 and January 2024. The efforts have evolved from the early work considering a broad swath of reference lab data elements to a much narrower set that were found to have consensus between multiple labs. 
+
+The explicit goals of the January connectathon included:
```

Comment:
### Implementation / Validation Venues
The decisions and consensus reached by the GenomeX Data Exchange community through the described process have been extensively tested and validated at HL7 FHIR Connectathons, including the May 2023 Clinical Genomics track (Connectathon 33) and the January 2024 Clinical Genomics - CodeX (GenomeX) track (Connectathon 35). Please refer to the following links for detailed information:

- [HL7 FHIR Connectathon 33 Clinical Genomics - May 2023](https://confluence.hl7.org/display/FHIR/2023+-+05+Clinical+Genomics)
- [HL7 FHIR Connectathon 35 Clinical Genomics - CodeX (GenomeX) - January 2024](https://confluence.hl7.org/pages/viewpage.action?pageId=204279063)

Reply:
@sirkyp - I think we can keep this in for now because it's just for our GenomeX members and serves as a complement to a GoogleDoc for now. We can remove this if it ever goes to at least an informative ballot.